ansible/runload.yaml

```diff
@@ -4,19 +4,20 @@
   hosts: loadrunners
   vars:
     ansible_host_key_checking: false
-    endpoints: ws://127.0.0.1:26656/websocket # comma separated list of endpoints
+    validator_0_ip: "{{ groups['validators'][0] }}"
+    validator_0_internal_ip: "{{ hostvars[validator_0_ip].internal_ip }}"
```

Comment:
Why should this expression have to go through `hostvars` to get at `validator_0_ip`...

Reply:
Right. I removed the two variable definitions and now just access directly the value I want in `hostvars`.